openpathsampling/pathmover.py

```diff
@@ -2338,8 +2322,13 @@ def __init__(self, **kwargs):
         super(MoveDetails, self).__init__(**kwargs)
 
 
+# leave this for potential backwards compatibility
 class SampleDetails(Details):
     """Details of a sample
+
+    Notes
+    -----
+    Deprecated. Will not present in future versions
```

Comment:
Thanks. Yes, changed it to be deprecated now and be removed in 2.0. We are not using it for analysis anymore and it is not needed. But if you remove it, loading of an old file will fail.